laravel-app/app/Http/Controllers/HomeController.php

```diff
@@ -16,6 +14,10 @@ public function index()
         return view('home');
     }
 
+    /**
+     *
+     * @return \Illuminate\Http\RedirectResponse
+     */
     public function logout()
```

Comment:
Since they are using Laravel 10. Laravel already removed doc block and used return type declaration.

Docs:
https://laravel.com/docs/10.x/releases#types
https://www.php.net/manual/en/language.types.declarations.php

Demo:
https://www.reddit.com/r/laravel/comments/11328e0/quick_demo_of_the_laravel_10x_shift/

Reply:
but are we using laravel 10 tho??